cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/StreamHandlerTest.java

```diff
@@ -33,28 +34,33 @@
 import com.google.common.reflect.TypeToken;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import org.apache.http.HttpResponse;
 import org.jboss.netty.handler.codec.http.HttpMethod;
 import org.jboss.netty.handler.codec.http.HttpResponseStatus;
+import org.junit.After;
 import org.junit.Assert;
 import org.junit.Test;
 
 import java.io.IOException;
 import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
 import java.net.URL;
 import java.util.List;
 
 /**
  * Test stream handler. This is not part of GatewayFastTestsSuite because it needs to start the gateway multiple times.
  */
-public class StreamHandlerTestRun extends GatewayTestBase {
+public abstract class StreamHandlerTest extends GatewayTestBase {
```

Comment:
Also, why did this get renamed and removed from GatewayTestsSuite? Want them to run outside the suite?